custom/icds/templates/icds/partials/location_reassignment.html

```diff
@@ -14,22 +14,4 @@
       </div>
     </div>
   </li>
-  <li>
-    <p>
-      {% blocktrans %}
-        Select a location type. The downloaded file will include all locations at the selected
-        level and all higher levels.
-      {% endblocktrans %}
-    </p>
-    <div class="row">
-      <div class="col-sm-4">
-        <select class="form-control hqwebapp-select2" data-bind="value: location_type">
```

Comment:
I think you can also remove `hqwebapp/js/widgets` from the js file.

Reply:
https://github.com/dimagi/commcare-hq/pull/26951/commits/0a4c463c2b86491b71e932f471b9888b619830a5